api/types/http.go

```diff
@@ -99,7 +99,7 @@ type NodeTypesListResponse struct {
 type ExecCreateParams struct {
 	Name         string         `json:"name,omitempty"`
 	Provider     Provider       `json:"provider"`
-	NodeTypeID   string         `json:"nodeTypeID,omitempty"`
```

Comment:
This will impact the CLI since it uses the same types.

Reply:
got it so I probably bonked the 'select gpu' flag, will go through and test

Reply:
see : https://github.com/unweave/cli/pull/13